tools/tertiary-analysis/pyscenic/pyscenic_ctx.xml

```diff
@@ -0,0 +1,124 @@
+<tool id="pyscenic_ctx" name="PySCENIC CTX" profile="21.09" version="0.12.1+galaxy0">
+    <description>
+        computes active regulons based on a gene regulatory network
+    </description>
+    <requirements>
+        <container type="docker">
+            aertslab/pyscenic:0.12.1
+        </container>
```

Comment:
This, the version, the profile, the citation should all move to a macro file.

Reply:
hehehe... beyond my LLM lazyness.

Reply:
Done now, thanks!